doc/users/explain/interactive.rst

```diff
@@ -170,7 +170,7 @@ and run the GUI event loop for the specified period of time.
 The GUI event loop being integrated with your command prompt and
 the figures being in interactive mode are independent of each other.
 If you use `pyplot.ion` but have not arranged for the event loop integration,
-your figures will appear but will not be interactive while the prompt is waiting for input.
+your figures will appear, but will not be interactive while the prompt is waiting for input.
```

Comment:
```suggestion
your figures will appear but will not be interactive while the prompt is waiting for input.
```

I do not think there should not be a comma here.

Reply:
I agree the comma is wrong here.   On the other hand, I think a sentence with more than one "but" is confusing, maybe: 
```suggestion
If you try to use `pyplot.ion` without arranging for the event-loop integration,
your figures will appear but will not be interactive while the prompt is waiting for input.
```

Reply:
I will change I think is better